### PR TITLE
install routine: template to html5

### LINKED ISSUE #3643: [Feature] use HTML5 tags
**Is your feature request related to a problem? Please describe.**
I would like to read the feeds with the Browser reader
Firefox Reader View: https://support.mozilla.org/en-US/kb/firefox-reader-view-clutter-free-web-pages

**Describe the solution you'd like**
One of the reasons is, that the HTML5 tags <header> <main> <footer> and <nav> are not used.

**Additional context**
It could also help for users with a screen reader to navigate through the page

### PR DESCRIPTION
**Pull request by math-GH**
Closes #3643 (forget to check the install routine within #3651) 

Changes proposed in this pull request:
- inserted `<main>`, `<nav>`, `<header`

How to test the feature manually:
1. run the install routine

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: app/install.php [hunk 1]
```diff
@@ -662,22 +662,24 @@ function printStep5() {
...
+			<li class="nav-header"><?= _t('install.steps') ?></li>
+			<li class="item<?= STEP == 0 ? ' active' : '' ?>"><a href="?step=0"><?= _t('install.language') ?></a></li>
+			<li class="item<?= STEP == 1 ? ' active' : '' ?>"><a href="?step=1"><?= _t('install.check') ?></a></li>
+			<li class="item<?= STEP == 2 ? ' active' : '' ?>"><a href="?step=2"><?= _t('install.bdd.conf') ?></a></li>
+			<li class="item<?= STEP == 3 ? ' active' : '' ?>"><a href="?step=3"><?= _t('install.conf') ?></a></li>
+			<li class="item<?= STEP == 4 ? ' active' : '' ?>"><a href="?step=4"><?= _t('install.this_is_the_end') ?></a></li>
+		</ul>
+	</nav>
 
 	<div class="post">
```
Comment (Frenzie) [inline app/install.php:684]: Is #global the main or this one?

**Review by Frenzie — APPROVED** [review]
lgtm

File: app/install.php (hunk 1 above)
Comment (math-GH) [reply app/install.php:684]: both would be fine.

#global would be better in sync to https://github.com/FreshRSS/FreshRSS/blob/edge/app/views/index/global.phtml (where the class `global` is changed to `<main>`)

Comment (Frenzie) [reply app/install.php:684]: Alright, let's stick to #global then.

**Pull request merged**